openfisca_france/parameters/impot_revenu/bareme_ir_depuis_1945/bareme.yaml

```
@@ -1443,6 +1451,9 @@ metadata:
       href: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000046860759/2024-01-01/
     - title: Loi 2023-1322, art. 2 du 29/12/2023 (LF pour 2024)
       href: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000048727355
+    2024-01-01:
+    - title: Impôt sur le revenu : tranches et taux d'imposition 2025
+      href: https://www.service-public.fr/particuliers/actualites/A18045
```

```suggestion
    2024-01-01:
      title: Loi n°2025-127, art. 2 du 14 février 2025 de finances pour 2025 
      href: https://www.legifrance.gouv.fr/eli/loi/2025/2/14/ECOX2423405L/jo/texte
```
il faut privilégier les textes de loi, qui font seuls foi